torch/_dynamo/variables/lists.py

```diff
@@ -152,6 +152,14 @@ def call_method(
             assert len(args) == 1
             assert not kwargs
             return _listlike_contains_helper(self.items, args[0], tx, options)
+        elif name == "index":
+            from .builder import SourcelessBuilder
+
+            assert len(kwargs) == 0
+            assert len(args) > 0 and len(args) <= 3
```

Comment:
I don't think you need these asserts now, the inliner will deal with it.

Reply:
true, but I'll leave it in for clarity of contract.

Reply:
If so, then you need to transform these into something that is not an assert. The assert here says we did something wrong, but user code can intentionally pass incorrect arguments to index, we shouldn't say we messed up here, it's the user's fault.